_pages/en/it-picture-long-term.md

```diff
@@ -38,59 +45,47 @@ They can also use the digital services on behalf of clients.
-> > ––The Honourable Jean-Yves Duclos, Minister of Families, Children and Social Development
+Ecosystem of services are created by leveraging private sector expertise where opportunities exist (win-win public-private partnerships), e.g.:
+- Banks detecting missed paycheck, job websites targeting people who are job hunting...
+- Enhanced security through integration in private sector service (e.g., banks detecting fraudulent transactions)
 
-## Private Sector
+### Open Collaboration
 
-Any platform or device can be integrated with GC and ESDC services.
-This is possible because ESDC services are available as APIs that can be used by third party platforms and services (Banks, CRA, Social media, Job boards, Internet of things, ..).
-
-- Create ecosystem of services by leveraging private sector expertise where opportunities exist (win-win public-private partnerships)
-  - E.g., banks detecting missed paycheck, job websites targeting people who are job hunting
-  - Enhanced security through integration in private sector service (e.g., banks detecting fraudulent transactions)
+- All of ESDC's custom built software are published as open source, allowing Academia and learning institutions to use them, helping grow technological skill sets in the country, fostering economic growth, improving transparency and trust in government, and increasing the breadth of talent for ESDC to use by welcoming contributions back
+- ESDC data (performance and automated decision making statistics) published as open datasets
 
 ## IITB-ESDC
 
 ### Modern Open Organization
 
 IITB is a modern open organization with a very open culture.
 Wherever possible code and projects are developed in the open and use of open source software is the norm.
+IITB is the digital, IM and IT advisor and enabler for ESDC and part of major business decisions.
 
-### Strong culture of learning
-
-TODO: Split in two (IITB and Partners/Clients)
-
-- IITB is the digital, IM and IT advisor and enabler for ESDC
-- IITB is part of major business decisions
 - 90% of employees are able to work from anywhere (TODO: validate measure with trends)
-- IITB is a modern open organization with a very open culture that values learning (TODO: find a tangible example)
 - All internal and external services are available through APIs
 - 98% of services are running on public cloud infrastructure (TODO: find cloud adoption rates)
-
-## Government Partners
-
-For:
-- Digital Exchange (APIs, Messaging)
-- Cloud Brokering
-- HR Services
-
-## Terms 
-
-ESDC: Employment and Social Development Canada
-IoT: [Internet of Things](https://en.wikipedia.org/wiki/Internet_of_things)
-IRCC: Immigration, Refugees and Citizenship Canada
+- All workloads that are classified Protected B and below are hosted on the public cloud.
+  - Classified workloads (secret information) on SSC data centres
```

Comment:
I wouldn't say that ... in 10 years, a lot may change. Full cloud ... many new solutions will exist in 10 years time

Reply:
I don't know, even if containers are old and it's now quantum serverless mesh orchestration, if it's still someone else's infrastucture, it's public cloud.

Reply:
Oh, right. When I said that, I meant to refer to 'SSC data centres' 

Perhaps... 
  - Classified workloads (secret information) on SSCs managed and approved networking solution